rep-2002.rst

```diff
@@ -0,0 +1,137 @@
+If a package is prevented from being automatically released into the stable distribution due to a dependency being unavailable in the stable distribution, that package may be automatically re-bloomed when the dependency becomes available in the stable distribution.
+
+
+Updating the rolling release platforms
+--------------------------------------
+
+The rolling release is intended to remain availabe indefinitely and so will not be retired or reach end-of-support status without an update to this REP and the ROS distribution release process.
+However, this does mean that platform changes during the life of the rolling distribution are going to occur and the rolling release may target platforms that are themselves unstable or in a pre-release state.
+When a platform change is coming, the intended date of change will be announced ahead of time.
+Once the platforms have been updated, everything currently in the rolling release will be automatically re-released via bloom to target the updated release platforms.
+
+
+Bootstrapping the rolling release
+---------------------------------
+
+The first release of the rolling rosdistro will use the same procedure we've used for all stable rosdistros previously.
+Packages will be released via bloom into an empty distribution until the distribution is relatively complete.
+Package maintainers may then choose to make releases into the rolling rosdistro.
+
+
+Recommendations for package maintainers
+=======================================
+
+Versioning between rolling and stable distributions
+---------------------------------------------------
+
+When a new stable release is created, the ROS core team strongly recommends that the next release made into the rolling release bump at least the minor version number.
+For example if the version of a package in both the rolling and current stable distribution is ``1.2.3`` and there are new changes that are targeting the rolling distribution only, bump the version to at least ``1.3.0``.
+This allows the patch release of the ``1.2.x`` series to be used for any future changes targeting only the stable distribution.
+Even if no changes are planned it is still a good idea to leave room for unanticipated fixes.
+The alternative is that version numbers may get confusing as the rolling distribution contains ``1.2.4`` but ``1.2.5`` is released only into the stable distribution as a fix on ``1.2.3``.
+
+
+Implementation
+==============
+
+
+Changes to rosdistro distribution files
+---------------------------------------
+
+A new distribution_status semantic value of ``rolling`` will be added to the rosdistro index v4 format specified in REP-153 [1]_.
+Rolling releases will be excluded from consideration in certain status pages (such as the blocked_releases_* pages) and considered differently from the active stable distributions.
```

Comment:
It would probably make sense to simply have the blocked releases analysis look at what can be made available in the specific distros from the rolling distro. We won't need to have it compare to the older ones.